model_utilities/config.ini

```
@@ -23,8 +23,14 @@ weight_decay = 1e-3
 scale_parameter = False
 relative_step = False
 
+[DATASET WEIGHTS]
```

```suggestion
[DATA WEIGHTS]
```
Or change `[DATA]` in the config to `[DATASETS]`.